gordon/main.py

```diff
@@ -166,7 +167,8 @@ def run(config_root):
     if plugin_names:
         logging.info(f'Loaded {len(plugin_names)} plugins: {plugin_names}.')
 
-    msg_router = _setup_message_router(plugins, **channels)
+    route_config = config.get('core', {}).get('route', {})
```

Comment:
How you have the `GordonRouter` initialization right now expects this to be set. Should we set a default? IMO it might be a good idea, like the general happy path as default.

Reply:
I liked `.* -> 'cleanup' (with default to drop message)` as the default, which because of your above comment I'm now realizing doesn't work like I thought it did (for whatever we go with, I will add a test this time!). 

I'm less inclined to put a default happy path in the router itself, just because that seems very implementation-specific. If we do, which would you want `consume->enrich->publish->cleanup` or `consume->publish->cleanup`? And should configured routes merge with the default or do a full overwrite?

Reply:
I think both could be the default path, and that it could be deduced if there's an `enrich` phase plugin, much like how we have currently. What does @matt0bi think about default path(s)?